scripts/run-in-pipeline.sh

```diff
@@ -0,0 +1,194 @@
+#
+#    FAIL_ON_LIVE : (True/False) Sets the condition to fail audit if there are verified live secrets found
+#      * Default:  True
+#
+#    FAIL_ON_UNAUDITED : (True/False) Sets the condition to fail audit if there are unaudited potential secrets found
+#      * Default:  True
+#
+#    FAIL_ON_AUDITED_REAL : (True/False) Sets the condition to fail audit if there are audited and set to real secrets found
+#      * Default:  True
+#
+# Example manual docker run within packaged container (git-defenders/detect-secrets-ubi):
+#    docker run  --env BASELINE=.secrets.baseline  --env FAIL_ON_LIVE=False  -it -a stdout --rm -v $(pwd):/code git-defenders/detect-secrets-ubi
+##
+
+## Constants for FAIL_ON_xx Environment Varibles
+_false="false"
+_true="true"
+
+## Setting Baseline filename, skip_scan, and fail-on-xx Boolean Defaults
+_baseline_default=.secrests.baseline
+_skip_scan_default=$_false
+_json_default=$_false
+_omit_instructions_default=$_false
+_fail_live_default=$_true
+_fail_unaudited_default=$_true
+_fail_audited_real_default=$_true
+
+## Constants representing Detect Secrets audit -reports --fail-on-xx paramenter options
+_fail_live_option="--fail-on-live"
+_fail_unaudited_option="--fail-on-unaudited"
+_fail_audited_real_option="--fail-on-audited-real"
+_omit_instructions_option="--omit-instructions"
+_json_option="--json"
+
+## Reading input Environment Variables while setting defaults for missing Environment Variables
+baseline=${BASELINE:=.$_baseline_default}
+json=${JSON:=${_json_default}}
+omit_instructions=${OMIT_INSTRUCTIONS:=$_omit_instructions_default}
+skip_scan=${SKIP_SCAN:=$_skip_scan_default}
+fail_live=${FAIL_ON_LIVE:=$_fail_live_default}
+fail_unaudited=${FAIL_ON_UNAUDITED:=$_fail_unaudited_default}
+fail_audited_real=${FAIL_ON_AUDITED_REAL:=$_fail_audited_real_default}
+
+##
+# Declare normalize function for normalizing the user input for Boolean vales to either true or false
+#   - This function allows the user input to be case-insensitive (ie TRUE, true, True, TrUe are all = true)
+#   - This function will return the default value for input if input represents any value other than true or false
+#     Note:  defaulting for missing Env Var input is handled before this function gets called
+#
+#   Inputs:
+#     - raw user input value for skip_scan or fail-on-xx parameter
+#     - default value for that same parameter
+#   Output:
+#     - case-insentive value (true/false) or default value
+##
+function normalize {
+    local user_param=$1
+    local default_val=$2
+
+    if [[ "${user_param,,}" == "$_true" ]]  # Note:  ${var,,} means to lowercase of var
+    then
+        result="$_true"
+    elif [[ "${user_param,,}" == "$_false" ]]
+    then
+        result="$_false"
+    else
+        result="$default_val"
+    fi
+
+    echo $result
+}
+
+## Initialize Detect Secrets audit_report parameter string
+audit_report_params=' '
+
+##
+# Starting the pipeline Detect Secrets run
+##
+echo "[ Starting Detect Secrets run ]"
```

Comment:
The DS reporting feature outputs a newline at the end, so there will be a newline between that output and the ending string. For consistency, it would be good to add a newline after the start string as well.
```suggestion
echo "[ Starting Detect Secrets run ]"
echo
```
Before:
<img width="571" alt="Screen Shot 2022-03-18 at 10 18 16 AM" src="https://user-images.githubusercontent.com/12804086/159024005-bf180958-fa19-4666-a56d-8fea1819ee34.png">
After:
<img width="564" alt="Screen Shot 2022-03-18 at 10 41 50 AM" src="https://user-images.githubusercontent.com/12804086/159024114-e1840e5b-46be-4347-95b6-2ba2bea2bbe8.png">

Reply:
jk when the JSON flag is included, no newlines are outputted in the report since the output would be pure json

Reply:
because of that, it would be good to also add a newline before the ending DS run string is printed. I'll add a suggestion for that.